src/core_plugins/kibana/public/context/query_parameters/actions.js

```diff
@@ -43,7 +46,13 @@ export function QueryParameterActionsProvider() {
     )
   );
 
+  const addTermsFilter = (state) => (field, values, operation) => {
```

Comment:
"termsFilter" is a bit of an overloaded (ahem) term since there's an actual Elasticsearch query type called "terms". Is there something you were trying to convey with this name that "addFilter" doesn't?